src/main/java/com/extrahardmode/ExtraHardMode.java

```diff
@@ -79,14 +77,27 @@ public void onEnable()
         // Register modules
         registerModule(RootConfig.class, new RootConfig(this));
         registerModule(MessageConfig.class, new MessageConfig(this));
+        
+        //Disable tutorial if it is disable in all worlds.
+        boolean dtut = true;
+        for (World world : getServer().getWorlds())
```

Comment:
To simplify things, I'd move this foreach loop to a separate method that accepts a RootNode enum, and returns true or false. Then you have less copy-pasted code.